cloud/datasync/package.json

```diff
@@ -35,10 +35,12 @@
     "@types/proxyquire": "^1.3.27",
     "del-cli": "^1.0.0",
     "mocha": "^3.4.2",
+    "mocha-typescript": "^1.1.5",
```

Comment:
more precisely [this line](https://github.com/feedhenry-raincatcher/raincatcher-core/blob/master/templates/javascript-usage/test/index.js#L9) calls into the TS test suite